plugins/net_plugin/net_plugin.cpp

```diff
@@ -3816,18 +3828,18 @@ namespace eosio {
       error_code ec;
 
       if (!ca.empty()){
-         fc_dlog(logger, "using verify file: ${p}", ("p", ca));
+         dlog("using verify file: ${p}", ("p", ca));
```

Comment:
if you don't have a special reason for this fc_dlog to dlog changes and the ones below, please revert them

Reply:
on some reason I don't see those lines in logs if `fc_dlog` is used. not sure why, need to investigate.

Reply:
That is because the `logger` is not initialized until plugin_startup and these are called in plugin_initialize

Reply:
so that sounds like you should use dlog then